views/agency-requests/edit_participant.php

```diff
@@ -161,43 +159,55 @@ function updateInsert( $db, $formid){
 		}
 		
 		
-		if($error == 0){
-		?>
-			
-			<div class="alert alert-success alert-dismissible fade show" role="alert">
-			<button type="button" class="close" data-dismiss="alert" aria-label="Close">
-			<span aria-hidden="true">&times;</span>
-			</button>
-				<strong>Success: </strong> Participant information has updated 
-			</div>
-			<div class="">
-				<span class="cpca btn"> 
-				<i class="fa fa-arrow-left"></i><a style="text-decoration:none; color:white;"href="/ps-view-participant/<?=$peopleid?>">
-				 Back to View Participant</a>
-				</span>
-			</div>
-			
-			  <?php
-		}
 	}
 	if(!empty($_POST['form'])){
 		updateInsert($db, $formid);
 	}
 	
-	//Currently waiting on stored procedure
-	/*$addressid = $_POST['addressid'];
-	$streetnumber = $_POST['house-update'];
+                                     
+	$addressid = $_POST['addressid'];
+	$streetnumber =$_POST['house-update'];
 	$streetaddress = $_POST['street-update'];
 	$aptinfo = $_POST['apt-update'];
 	$city = $_POST['city-update'];
-	$state = $_POST['street-update'];
+	$state = $_POST['state-update'];
 	$zip = $_POST['zip-update'];
-	
-	$addressupdate = $db->query("UPDATE addresses SET addressnumber = $2, aptinfo = $3, street = $4, zipcode =$5 WHERE addressid = $1",
-								[$addressid, $streetnumber, $aptinfo, $streetaddress, $zip]);*/
+
+                                                    
+    $res = $db->query('SELECT addressUpdate(
+                        pID := $1::INT,
+                        newAddressNumber := $2::INT,
+                        newAptInfo := $3::TEXT,
+                        newStreet := $4::TEXT,
+                        newZipcode := $5::VARCHAR(5),
+                        newCity := $6::TEXT,
+                        newState := $7::STATES)',array($peopleid, $streetnumber, $aptinfo, $streetaddress, $zip, $city, $state));
+    if(!$res){
+        $error++;
+    }
+    
+    if($error > 0){
+        $notification = new Notification('Participant Update Failed','There were issues in submitting the participant updates.','danger');
+        $notification->display();
+    }else{
+        $notification = new Notification('Participant Update Success','Participant information has been updated.','success');
+        $notification->display();
+        ?>
+        <script>window.location = "/ps-view-participant/<?=$peopleid?>";
+        </script>
+        <?php
+        
+    }
+                     
 }
 
```

Comment:
Like right here would be a good place for the header. You shouldn't get the error after that.